### PR TITLE
Card bug squish

### PR DESCRIPTION
**Pull request by KeystrokeCascade**
Images now centered
Long text strings now ellipse
Fixed small overlap in single-column and double column layout

### CODE REVIEW

File: src/app/components/horse-books-listing/horse-book-list-item/horse-book-list-item.component.scss [hunk 1]
```diff
@@ -21,19 +19,28 @@ mat-card{
 	text-align: middle;
 }
 .image img{
+	padding-top: 10%;
```
Comment (tsitra) [inline src/app/components/horse-books-listing/horse-book-list-item/horse-book-list-item.component.scss]: Is this necessary for the fix?

Comment (KeystrokeCascade) [reply src/app/components/horse-books-listing/horse-book-list-item/horse-book-list-item.component.scss]: the padding-top? That was in there before.

Comment (tsitra) [reply src/app/components/horse-books-listing/horse-book-list-item/horse-book-list-item.component.scss]: This is a diff between master and this branch.

File: src/app/components/horse-books-listing/horse-book-list-item/horse-book-list-item.component.scss [hunk 2]
```diff
@@ -21,19 +19,28 @@ mat-card{
...
 	text-overflow: ellipsis;
 	overflow: hidden;
 	white-space: nowrap;
-	display: inline-flex;
+	text-align: center;
 }
 
 
-@media only screen and (max-width: 679px) {
+@media only screen and (max-width: 696px) {
```
Comment (tsitra) [inline src/app/components/horse-books-listing/horse-book-list-item/horse-book-list-item.component.scss]: This shows you changing the width that the single-column stuff kicks in at from 679 to 696.

Comment (tsitra) [reply src/app/components/horse-books-listing/horse-book-list-item/horse-book-list-item.component.scss]: Per our discussion in Discord, the list is 1 column starting at 696px, so this is an appropriate change. Good catch!

**Comment by KeystrokeCascade** [comment]
Without top padding, text is now clipping with the image *just*.  Add it back in?

**Comment by tsitra** [comment]
> Without top padding, text is now clipping with the image _just_. Add it back in?

The touching of the text and image should be resolved through the title - we want to make sure the title has enough space around it from ANY element, not that the image has X space from the element preceding it.

**Comment by tsitra** [comment]
Resolves #29

**Pull request merged**